geode-core/src/main/java/org/apache/geode/management/internal/operation/OperationState.java

```diff
@@ -28,12 +28,25 @@
  */
 public class OperationState<A extends ClusterManagementOperation<V>, V extends OperationResult>
     implements Identifiable<String> {
+  private static final long serialVersionUID = 8212319653561969588L;
   private final String opId;
   private final A operation;
   private final Date operationStart;
   private Date operationEnd;
   private V result;
   private Throwable throwable;
+  private String locator;
```

Comment:
Can this be DistributedID than a String ID. That way we can avoid converting to string in other places?

Reply:
Do you mean `InternalDistributedMember` when you talk about `DistributedID`? I was using the `InternalDistributedMember` as the type for `locator` field. But it is a large object with many fields. We only need to identify the locator, so a String type of a member's ID should be good.